blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxIO.java

```diff
@@ -172,12 +211,26 @@ public void onTimerRemoved(String name) {
         });
     }
 
+    @Override
     public void close() { //Not to be used with time-series data.
         session.close();
```

Comment:
Not sure if we need null check here as session/cluster is not guaranteed to be initialized when class is loaded.

Reply:
`cluster` and `session` are final and set by the constructor. There's no way they could be null by the time you could call `close()`. Actually, things would crash before you ever got that far because `cluster` is used by the monitoring and logging stuff during the constructor call. If it's null, it fails then. I saw it happen while I was working this change out.